### ISSUE TITLE
Authentication page needs clarification 

### ISSUE THREAD

**Issue by Tejeev** [rancher/rancher#27614]
**What kind of request is this (question/bug/enhancement/feature request):**
RFE

**Steps to reproduce (least amount of steps as possible):**
In Global view, go to Security > Authentication
Test authentication with a user

**Result:**
Testing a user does not result in a pass/fail notification if it succeeds.
There is no indication that the user you use to "Authenticate with Active Directory," will be made the Rancher LDAP auth Admin and will be granted full admin privileges over Rancher.
The edit button is still available but does nothing once it is selected, and you are in edit mode.
In conclusion, the Authentication page is generally obtuse, buggy, and needs review.

**Other details that may be helpful:**
I find that this has impacted several users and the UI for Auth would benefit from an update to enhance UI experience.

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): v2.3.6 - v2.4.4

**Comment by aquavitae** [comment]
I have also found this page to be very confusing and non-inuitive. Before signing up for support with Rancher I spend a good few hours on calls trying to understand how the Auth worked, and eventually just gave up when login and permissions seemed to be correct for what we wanted.
